LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DownlinkMessageBuilder.cs

```diff
@@ -385,9 +373,9 @@ private static ushort ValidateAndConvert16bitFCnt(uint fcntDown)
                         case Cid.Zero:
                         case Cid.One:
                         case Cid.LinkADRCmd:
-                            if (rxpk != null)
+                            if (loRaRequest != null)
```

Comment:
FYI, noticed this code is a duplicate of the above case that I consolidated in #1130.